cirrus/google_cloud/manager.py

```diff
@@ -306,6 +307,60 @@ def get_project_organization(self):
             org = info["parent"]["id"]
         return org
 
+    def get_project_members(self):
+        """
+        Get all the members given a project
+
+        Returns:
+            list(str): list of members
+
+            .. code-block:: python
+
+            {
+              "version": 1,
+              "etag": "BwVvrr5i9Jc=",
+              "bindings": [
+                {
+                  "role": "roles/compute.serviceAgent",
+                  "members": [
+                    "serviceAccount:service-xxxx@compute-system.iam.gserviceaccount.com"
+                  ]
+                },
+                {
+                  "role": "roles/container.serviceAgent",
+                  "members": [
+                    "serviceAccount:service-xxxx@container-engine-robot.iam.gserviceaccount.com"
+                  ]
+                },
+                {
+                  "role": "roles/owner",
+                  "members": [
+                    "user:test@planx-pla.net"
+                  ]
+                }
+              ]
+            }
+        """
+        members = []
+
+        api_url = _get_google_api_url(
+            "projects/" + self.project_id + ":getIamPolicy", GOOGLE_CLOUD_RESOURCE_URL)
+        response = self._authed_request("POST", api_url)
+        if response.status_code != 200:
+            raise Exception('Unable to get IAM policy for project {}. The status code is {}'.format(
```

Comment:
Let's create a new Error type in `cirrus.google_cloud.errors` called GoogleApiError instead of using the generic `Exception`

Reply:
fixed at 20b0fe69f63d83f478367e6011638d17930e7e62